src/requests/domainAliases/models.ts

```diff
@@ -0,0 +1,99 @@
+export interface CreateNewDomainAliasBodyParameterModel {
+  /**
+   * Domain Alias
+   */
+  alias: string;
+  /**
+   * Domain name this Alias applies to
+   */
+  domain: string;
+}
+
+export interface GetAliasInfoResponseModel {
+  /**
+   * Indicates successful response
+   */
+  success: boolean;
+  /**
+   * ID of the Alias
+   */
+  id: string;
+  /**
+   * Alias domain
+   */
+  alias: string;
+  /**
+   * Alias target
+   */
+  domain: string;
+  /**
+   * <date-time>
+   * Datestring of the time the alias was created
+   */
+  created: string;
+}
+
+export interface ListRegisteredDomainAliasesQueryParametersModel {
```

Comment:
The data in here looks awfully similar to data we can extend (src/models/index.ts) ;)